kitsune/search/api.py

```diff
@@ -54,7 +54,8 @@ def _question_suggestions(searcher, text, locale, product, max_results):
         model='questions_question',
         question_is_archived=False,
         question_is_locked=False,
-        question_has_helpful=True)
+        question_has_helpful=True,
```

Comment:
I see no upside to filter for solved threads that also have helpful votes, so we should get rid of this filter.